teaspoon-jasmine/spec/integration_spec.rb

```diff
@@ -0,0 +1,149 @@
+require_relative "./spec_helper"
+
+feature "Running in the browser", browser: true do
+  describe "with jasmine1" do
+    before do
+      path = File.expand_path('../../spec/javascripts', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.